Sources/TuistGraph/Models/SwiftPackageManagerDependencies.swift

```diff
@@ -10,4 +10,58 @@ public struct SwiftPackageManagerDependencies: Equatable {
     public init(_ packages: [Package]) {
         self.packages = packages
     }
+
+    /// Returns `Package.swift` representation.
+    ///
+    /// **NOTE** It is a temporary solution until Apple resolves: https://forums.swift.org/t/pitch-package-editor-commands/42224
+    public func stringValue() -> String {
+        """
+        // swift-tools-version:5.3
```

Comment:
What about using the same Swift version the environment is pointing to. You can use [this method](https://github.com/tuist/tuist/blob/main/Sources/TuistSupport/System/System.swift#L165) to get it.

Reply:
I have different versions for `Swift` and `Swift tools`.

<img width="514" alt="Zrzut ekranu 2021-03-15 o 21 33 43" src="https://user-images.githubusercontent.com/4774319/111217767-372b1d00-85d6-11eb-9c58-a360ed711fce.png">

It end ups with

<img width="1365" alt="Zrzut ekranu 2021-03-15 o 21 35 35" src="https://user-images.githubusercontent.com/4774319/111217909-62ae0780-85d6-11eb-9258-144ba2be5a67.png">

What do you think about adding new method `Systeming.swiftToolsVersion`?  @pepibumur @fortmarek